### PR TITLE
Fix `GUID_AGE` in `$METADATA`

### PR DESCRIPTION
**Pull request by catsuryuu**

### CODE REVIEW

File: drakrun/drakrun/drakpdb.py [hunk 1]
```diff
@@ -413,20 +415,21 @@ def make_pdb_profile(filepath, dll_origin_path=None, dll_path=None):
...
     )
-    symstore_hash = "%s%s" % (guid_str, pdb.STREAM_PDB.Age)
-    base_fn = os.path.splitext(os.path.basename(filepath))[0]
+    pdb_symstore_hash = "%s%x" % (pdb_guid_str, pdb.STREAM_PDB.Age)
+    base_filename = os.path.splitext(os.path.basename(filepath))[0]
 
     profile["$METADATA"] = {
-        "GUID_AGE": symstore_hash,
+        "DLL_GUID_AGE": dll_symstore_hash,
+        "PDB_GUID_AGE": pdb_symstore_hash,  # TOREVIEW: Is "PDB_GUID_AGE" ok or "GUID_AGE" should remain?
```
Comment (catsuryuu) [inline drakrun/drakrun/drakpdb.py:430]:
```suggestion
        "GUID_AGE": pdb_symstore_hash,
```

File: drakrun/drakrun/drakpdb.py [hunk 2]
```diff
@@ -413,20 +415,21 @@ def make_pdb_profile(filepath, dll_origin_path=None, dll_path=None):
...
+    pdb_guid = pdb.STREAM_PDB.GUID
+    pdb_guid_str = "%08x%04x%04x%s" % (
+        pdb_guid.Data1,
+        pdb_guid.Data2,
+        pdb_guid.Data3,
+        pdb_guid.Data4.hex(),
     )
-    symstore_hash = "%s%s" % (guid_str, pdb.STREAM_PDB.Age)
-    base_fn = os.path.splitext(os.path.basename(filepath))[0]
+    pdb_symstore_hash = "%s%x" % (pdb_guid_str, pdb.STREAM_PDB.Age)
```
Comment (chivay) [inline drakrun/drakrun/drakpdb.py:425]:
```suggestion
    pdb_symstore_hash = "%s%X" % (pdb_guid_str, pdb.STREAM_PDB.Age)
```

File: drakrun/drakrun/drakpdb.py [hunk 3]
```diff
@@ -413,20 +415,21 @@ def make_pdb_profile(filepath, dll_origin_path=None, dll_path=None):
...
-        guid.Data2,
-        guid.Data3,
-        guid.Data4.hex().upper(),
+    pdb_guid = pdb.STREAM_PDB.GUID
+    pdb_guid_str = "%08x%04x%04x%s" % (
+        pdb_guid.Data1,
+        pdb_guid.Data2,
+        pdb_guid.Data3,
+        pdb_guid.Data4.hex(),
     )
```
Comment (chivay) [inline drakrun/drakrun/drakpdb.py:424]: I think we could extract it into a separate function

Comment (catsuryuu) [reply drakrun/drakrun/drakpdb.py:424]: I planned refactoring on the next PR.

File: drakrun/drakrun/drakpdb.py (hunk 2 above)
Comment (catsuryuu) [reply drakrun/drakrun/drakpdb.py:425]: Why tho?

**Review by chivay — APPROVED** [review]

File: drakrun/drakrun/drakpdb.py (hunk 2 above)
Comment (chivay) [reply drakrun/drakrun/drakpdb.py:425]: https://github.com/google/rekall/blob/master/rekall-core/rekall/plugins/tools/mspdb.py#L930

**Pull request merged**